lib/dug.rb

```diff
@@ -10,7 +10,7 @@
 
 module Dug
   LABEL_RULE_TYPES = %w(organization repository reason state)
-  GITHUB_REASONS = %w(author comment mention team_mention state_change assign manual subscribed)
+  GITHUB_REASONS = %w(author comment mention team_mention state_change assign manual subscribed push)
```

Comment:
Is this true? I don't see `push` in the [API documentation](https://developer.github.com/v3/activity/notifications/#notification-reasons) _but_ I know that Github tweaks it's Notification API all the time and don't really change their docs, so I believe it.